serverless-plugin/index.js

```diff
@@ -282,19 +265,20 @@ class ServerlessPlugin {
             const matches = handlerPattern.exec(func.handler);
 
             if (!matches) {
-                throw new Exception(`handler ${func.handler} was not of the form 'packageName.executableName' or 'dir1/dir2/packageName.executableName'.`);
+                throw new Exception(`handler ${func.handler} was not of the form 'ModuleName.functionName' or 'dir1/dir2/ModuleName.functionName'.`);
             }
 
-            const [_, directory, packageName, executableName] = matches;
+            const directory = "";
+            const executableName = "haskell_lambda";
```

Comment:
The fact that the executable must be named this should be documented. Also, it's a bit redundant - the whole project is in Haskell, why put `haskell` in the executable name?

Reply:
It was a completely arbitrary decision, doesn't matter actually, the idea is that the user uses the template and forgets about the executable name, but you are right, it should be documented.